frontend/src/react/Layout/LayoutApp.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux';
+
+import CreateExam from '../Exam/CreateExam';
+import {getExams} from '../../redux/exams/actions/get';
+import {postExam} from '../../redux/exams/actions/post';
+
+class LayoutApp extends Component {
+
+    componentDidMount() {
+        this.props.fetchExams();
```

Comment:
Si vous voulez du méga swag, vous pouvez utiliser les Hooks à la place :P

Reply:
ça donne quoi avec des Hooks ? j'en ai entendu parler mais j'ai pas encore creusé le sujet ^^

Reply:
Regarde cette conférence https://youtu.be/dpw9EHDh2bM?t=689

Reply:
Merci, je regarderai ça :D